<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeadMesh</class>
 <widget class="QMainWindow" name="HeadMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_capture">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn On Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_end">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>510</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>END</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>661</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="system_select">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Brain 10-20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brain 10-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brain 10-5</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_recapture">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>320</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Recapture Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Mesh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
